packages/styles/scss/patterns/listsection/_listsection-item.scss

```diff
@@ -0,0 +1,55 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin listsection-item {
+    .#{$prefix}--listsection-item {
+      color: $text-01;
+      &__title {
+        @include type-style('expressive-heading-02');
+      }
+      &__content {
+        @include type-style('body-long-02');
+      }
+      &__link {
+        color: $link-01;
+        text-decoration: none;
+        outline: none;
+        display: inline-block;
+        border: solid 2px transparent;
+        @include type-style('body-short-02');
+        &:visited {
+          color: $text-02;
+        }
+        &:focus {
+          border-color: $focus;
+        }
+        &:hover {
+          text-decoration: underline;
+          cursor: pointer;
+        }
+        &__inner {
+          display: flex;
+          align-items: center;
+        }
+        &__icon {
+          fill: $link-01;
+          margin-left: $spacing-03;
+        }
+      }
+    @include carbon--breakpoint('sm') {
+      &__title {
+        margin-top: $layout-03;
+      }
+      &__link {
+        margin-top: $spacing-03;
```

Comment:
Is this supposed to be `16px`?

![image](https://user-images.githubusercontent.com/3793636/67693511-d0b94680-f96f-11e9-830f-c3573c08c758.png)

Reply:
$spacing-03 adds .5rem/8px margin top but it has 2px transparent border around link for focus state.
![Screen Shot 2019-10-29 at 1 17 01 PM](https://user-images.githubusercontent.com/8072521/67791807-7e028c00-fa4e-11e9-9650-be3808a8c358.png)

Reply:
That's still doesn't add up to 16px from the specs. I think all you need to do is use `$carbon--spacing-05`, the `2px` border should be part of the `Link` component.

Reply:
I am not sure  which specs you are talking about. It's .5rem/8px in my specs.

![Screen Shot 2019-10-29 at 7 57 51 PM](https://user-images.githubusercontent.com/8072521/67818177-a9a16880-fa86-11e9-9364-d20748ef7ca5.png)